pkg/client/eventsbuilder.go

```diff
@@ -32,14 +32,26 @@ func (builder *eventsBuilder) getStartPosition() *orderer.SeekPosition {
 
 type eventOption = func(builder *eventsBuilder) error
 
-// Checkpointer provides the current position for event processing.
-type Checkpointer interface {
+// Checkpoint provides the current position for event processing.
+type Checkpoint interface {
 	// BlockNumber in which the next event is expected.
 	BlockNumber() uint64
 	// TransactionID of the last successfully processed event within the current block.
 	TransactionID() string
 }
 
+// Checkpointer provides the current position for event processing.
```

Comment:
Checkpointer has different behaviour to Checkpoint. Perhaps:
```suggestion
// Checkpointer allows update of a checkpoint position after events are successfully processed.
```